src/app.ts

```diff
@@ -31,14 +31,13 @@ export class App {
       new ValidationPipe({
         whitelist: true,
         transform: true,
-        forbidNonWhitelisted: true,
+        forbidNonWhitelisted: true, // TODO APIM-128: Should we remove this setting?
```

Comment:
I didn't realise we had `forbidNonWhitelisted: true` for our global `ValidationPipe`. The `forbidNonWhitelisted: true` option means that `class-validator` will throw a validation error if there are extra fields in the request body / URL params / query string that the DTO class doesn't declare _when that body / param / query string uses the global pipe_.

For all our POST requests, we're overriding the global pipe for the request body anyway, so this setting is not used, but @AlexBramhill's #148 doesn't override the global pipe so I should ask him to change this setting in his PR

Reply:
I have put this on Alex's PR